py/objint.c

```diff
@@ -363,6 +363,10 @@ mp_int_t mp_obj_int_get_checked(mp_const_obj_t self_in) {
     return MP_OBJ_SMALL_INT_VALUE(self_in);
 }
 
+mp_uint_t mp_obj_int_get_uint_checked(mp_const_obj_t self_in) {
+    return MP_OBJ_SMALL_INT_VALUE(self_in);
```

Comment:
I'd probably lean to code reuse here and return mp_obj_int_get_checked so that if the implementation would change you only have to change it once. But I don't know if the compiler is smart enough to skip the extra call, nor if that matters.

Reply:
I don't think we should trust the compiler to always skip the extra call. I could introduce another helper function/macro to be used by both functions, but I don't think it matters that much.